### ISSUE TITLE
Allow viewing of images inside a buffer

### ISSUE THREAD

**Issue by Fuuzetsu** [yi-editor/yi#501]
With gtk+pango front-end, it should be possible to view images inside of the buffer. This could have very nice applications, such as when working with `diagrams`.

**Comment by zanzix** [comment]
Hey @Fuuzetsu, did you ever make much progress on this?

**Comment by Fuuzetsu** [comment]
No, I did no work there since January 2014 and it was nothing that I would recommend putting into code anyway, it was pretty awful way; basically it made a special buffer type for images and it would render the image inside of it, IIRC. I suppose the code still exists if you want to look at it but it's not useful: https://github.com/yi-editor/yi/commit/b8598670cb0a6f4d59ce178792ad484ff84418b0

**Comment by Fuuzetsu** [comment]
I randomly stumbled upon the results of the commit/branch I linked to above back from when I had it running in 2014. Here it is for reference.

![1390398252](https://user-images.githubusercontent.com/893115/29460278-48bd0ea0-841f-11e7-8885-0da494e798b8.png)
